packages/ses/src/compartment-evaluate.js

```diff
@@ -20,9 +20,10 @@ export const getCompartmentEvaluator = (compartmentFields, options) => {
   } = options;
 
   let safeEvaluate;
+  let scopeProxy;
 
   if (__moduleShimLexicals__ === undefined && !sloppyGlobalsMode) {
-    ({ safeEvaluate } = compartmentFields);
+    ({ safeEvaluate, scopeProxy } = compartmentFields);
```

Comment:
`scopeProxy` on `compartmentFields` now?

Reply:
Yeah #898 introduces a shared evaluator (saved on the `compartmentFields`), so in this change I simply save the `scopeProxy` that it uses.